### DISCUSSION TITLE
The primary color is specified, but the datePicker component does not reflect this.😭

### DISCUSSION THREAD

**Discussion by cksal0805** [ant-design/ant-design#40046] [Q&A]
Hi, I encountered a problem while currently migrating Antd v4 to v5.

The primary color is not applicable to the date picker.

```js
token: {
    colorPrimary: '#ed5ea8',
},
components: {
  DatePicker: {
      colorPrimary: '#ed5ea8',
      colorPrimaryActive: '#ed5ea8',
    },
}
```
<img width="270" alt="스크린샷 2023-01-05 오후 9 49 58" src="https://user-images.githubusercontent.com/69236698/210783975-c04f6c1d-3ee8-458c-8103-4e636543214a.png">

It's not working.

Help me please

**Comment by zombieJ** [comment]
Seems work as expect:

https://codesandbox.io/s/ji-ben-antd-5-1-2-forked-7z78g5?file=/demo.tsx

**Comment by Vezral** [comment]
Encountered the same problem while migrating from antd4 to antd 5.11.

Solved by updating `DatePicker` import from `antd/lib` to `antd`
